hybrasyl/Objects/User.cs

```diff
@@ -1762,6 +1763,32 @@ public User GetFacingUser()
             return (User)contents.FirstOrDefault(y => y is User);
         }
 
+        public Monster GetFacingMonster()

```

Comment:
as is, this is a duplicate of `Creature.GetFacingTargets` or `Creature.GetDirectionalTargets`

Reply:
removed.